pom.xml

```diff
@@ -249,16 +251,20 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>commons-logging</groupId>
```

Comment:
Conflict arose from hadoop-mapreduce-client-core. Added version 1.1.1 to resolve conflicts.

Reply:
Can you please add a comment in the code?